packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-editor.tsx

```diff
@@ -78,7 +79,8 @@ export const PipelineEditor: React.FunctionComponent<PipelineEditorProps> = ({
   serverVersion,
   onChangePipelineText,
 }) => {
-  const fields = useAutocompleteFields(namespace);
+  const connectionInfo = useConnectionInfo();
+  const fields = useAutocompleteFields(connectionInfo, namespace);
```

Comment:
We talked about making this hook use connectioninfo internally instead of manually updating every single use-case in the app. Are we not pursuing this anymore?